web/src/components/ScrollingSearchBox.tsx

```diff
@@ -22,20 +24,32 @@ const ScrollingSearchBox = ({
   placeholderText,
   spaceSearch = false,
 }: ScrollingSearchBoxProps): ReactElement => {
+  const [searchBarHeight, setSearchBarHeight] = useState(SEARCH_BAR_HEIGHT)
```

Comment:
🤔 I think this number is quite meaningless now, so I think we should just remove it and initialize with 0.
```suggestion
  const [searchBarHeight, setSearchBarHeight] = useState(0)
```